packages/slice-machine/server/src/api/index.ts

```diff
@@ -62,6 +63,17 @@ router.post(
   }
 );
 
+router.post(
+  "/tracking/onboarding",
+  async (req: express.Request, res: express.Response) => {
+    return onboarding(req.body)
```

Comment:
looking at how the other routes are made in this file, I think we should have the same.

Reply:
you mean with `await`?

Reply:
fine, but we'll lose some of the error handling in `packages/slice-machine/server/src/api/tracking/onboarding.ts` and also type checking of errors will be more tricky